Jenkinsfile

```diff
@@ -487,6 +485,102 @@ pipeline {
                         }
                     }
                 }
+                stage('Build on CentOS 7 debug') {
+                    when {
+                        beforeAgent true
+                        allOf {
+                            expression { ! skip_stage('build-centos7-gcc-debug') }
```

Comment:
This should also probably be skipped for Quick-build like the other "long" build stages are such as: https://github.com/daos-stack/daos/blob/f0bc27fd82f876af89a326875d13ada0d639af97/Jenkinsfile#L589

Reply:
But for quickbuild, they use QUICKBUILD_DEPS, no ?

Reply:
In this case, the presence of `Quick-build: true` should skip the stage like it does for all of the long-build stages, except of course the main CentOS 7 stage, which does use `QUICKBUILD_DEPS`.

Reply:
I don't get the purpose of skipping it if it can use QUICKBUILD_DEPS.   I thought the point of quick build was to build faster by skipping prerequisites, not to skip build stages altogether.  Skipping it means the build doesn't get tested at all which seems wrong to me.  The main purpose of this is to ensure that we aren't introducing new warnings and if we skip building altogether, doesn't that defeat the purpose?   Or is quickbuild only to be used during development phase of a patch and not for final ?

Reply:
So I guess it depends on your perspective of quick-build.

> Or is quickbuild only to be used during development phase of a patch and not for final ?

￼That's generally how I use it.  I revert to a full build when I am doing a run I expect to get reviewed and landed.

Now, if we wanted to change that and make quick-build the default, then yes, it shouldn't skip these stages but rather should apply the quick-build to them also.